modules/nw-metallb-advertise-ip-pools-from-node-subset.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+
+[id="nw-metallb-advertise-ip-pools-to-node-subset_{context}"]
+= Advertising an IP address pool from a subset of nodes 
+
+To advertise a pool of IP addresses from a specific set of nodes only, use the `.spec.nodeSelector` specification in the BGPAdvertisement custom resource. This specification associates a pool of IP addresses with a set of nodes in the cluster. This is useful when you have nodes on different subnets in a cluster and you want to advertise IP addresses from a specific subnet, for example a public-facing subnet only. 
```

Comment:
nit: the ip assigned from a given pool

Reply:
Edited this slightly, hopefully it is clearer now in the opening paragraph.